sdk/keyvault/azure-security-keyvault-certificates/src/keyvault_certificates_common_request.cpp

```diff
@@ -17,8 +17,10 @@ _detail::KeyVaultCertificatesCommonRequest::SendRequest(
     Azure::Core::Http::Request& request,
     Azure::Core::Context const& context)
 {
+  request.SetHeader("content-type", "application/json");
   auto response = pipeline.Send(request, context);
   auto responseCode = response->GetStatusCode();
```

Comment:
Can you declare these `const auto` ? IMHO, these are easier to read, knowing that you won't need to follow them being updated in the code below. In general, not in this place alone.